tests/test_client.py

```diff
@@ -1947,6 +1954,7 @@ def test_index_update(client, user):
     assert record["metadata"] == dataNew["metadata"]
     assert record["acl"] == dataNew["acl"]
     assert record["authz"] == dataNew["authz"]
+    assert record["description"] == dataNew["description"]
```

Comment:
Okay, kinda nitpicky but could you change the `dataNew` to `data_new` or something similar just to keep things the same format. We dont do camel case for most of our Python code